backend/pkg/api/data_access/vdb_summary.go

```diff
@@ -114,113 +113,123 @@ func (d *DataAccessService) GetValidatorDashboardSummary(ctx context.Context, da
 		BlocksScheduled        uint64          `db:"blocks_scheduled"`
 		SyncExecuted           uint64          `db:"sync_executed"`
 		SyncScheduled          uint64          `db:"sync_scheduled"`
+		MinEpochStart          int64           `db:"min_epoch_start"`
+		MaxEpochEnd            int64           `db:"max_epoch_end"`
 	}
 
-	wg.Go(func() error {
-		ds := goqu.Dialect("postgres").
-			Select(
-				goqu.L("ARRAY_AGG(r.validator_index) AS validator_indices"),
-				goqu.L("SUM(COALESCE(r.attestations_reward, 0) + COALESCE(r.blocks_cl_reward, 0) + COALESCE(r.sync_rewards, 0) + COALESCE(r.slasher_reward, 0)) AS cl_rewards"),
-				goqu.L("COALESCE(SUM(r.attestations_reward)::decimal, 0) AS attestations_reward"),
-				goqu.L("COALESCE(SUM(r.attestations_ideal_reward)::decimal, 0) AS attestations_ideal_reward"),
-				goqu.L("COALESCE(SUM(r.attestations_executed), 0) AS attestations_executed"),
-				goqu.L("COALESCE(SUM(r.attestations_scheduled), 0) AS attestations_scheduled"),
-				goqu.L("COALESCE(SUM(r.blocks_proposed), 0) AS blocks_proposed"),
-				goqu.L("COALESCE(SUM(r.blocks_scheduled), 0) AS blocks_scheduled"),
-				goqu.L("COALESCE(SUM(r.sync_executed), 0) AS sync_executed"),
-				goqu.L("COALESCE(SUM(r.sync_scheduled), 0) AS sync_scheduled")).
-			From(goqu.T(table).As("r")).
-			GroupBy(goqu.L("result_group_id"))
-
-		if len(validators) > 0 {
+	ds := goqu.Dialect("postgres").
+		From(goqu.L(fmt.Sprintf(`%s AS r FINAL`, clickhouseTable))).
+		With("validators", goqu.L("(SELECT dashboard_id, group_id, validator_index FROM users_val_dashboards_validators WHERE dashboard_id = ?)", dashboardId.Id)).
+		Select(
+			goqu.L("ARRAY_AGG(r.validator_index) AS validator_indices"),
+			goqu.L("SUM(COALESCE(r.attestations_reward, 0) + COALESCE(r.blocks_cl_reward, 0) + COALESCE(r.sync_rewards, 0) + COALESCE(r.blocks_cl_slasher_reward, 0)) AS cl_rewards"),
```

Comment:
```suggestion
			goqu.L("COALESCE(SUM(r.attestations_reward + r.blocks_cl_reward + r.sync_rewards + r.blocks_cl_slasher_reward), 0) AS cl_rewards"),
```
these columns arent nullable in clickhouse so this should work

Reply:
Changed.